src/main/java/it/gov/pagopa/pu/registry/model/BaseRegistry.java

```diff
@@ -0,0 +1,28 @@
+package it.gov.pagopa.pu.registry.model;
+
+import it.gov.pagopa.pu.registry.enums.EventType;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.springframework.data.annotation.Id;
+
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@SuperBuilder
+@EqualsAndHashCode
+@ToString
+public abstract class BaseRegistry {
+  @Id
+  private String eventId;
+  private EventType eventType;
+  private LocalDateTime eventDateTime;
```

Comment:
As you could see from PaymentEventDTO class, this is an OffsetDateTime, while the enum should not be re-created, but you should use the same used in PaymentEventDTO
```suggestion
  private OffsetDateTime eventDateTime;
```